### DISCUSSION TITLE
Clinical significance/ Pathogenic,. etc.

### DISCUSSION THREAD

**Discussion by DogCatSpace** [KarchinLab/open-cravat#308] [General]
When I began using O-Cravat, I had a dropdown menu (after Annotating, Filtering, and looking at output) under the Variant tab for clinical significance (Pathogenic, etc.).  This drop down disappeared; I went to the "store" and added every module under "Clinical Relevance" and the dropdown still does not appear, despite my clicking all annotation options before annotating.  

Am I correct in assuming this clinical significance tab only comes with one of the modules?

**Comment by jasminebro** [comment]
Hi @DogCatSpace. The variant tab only displays data compiled from the annotators chosen. You can view all of the features provided for each annotator by clicking the "+" sign next to the annotator name/header. A clinical significance tab/column can come with various annotators. 

For example, if you are using the AlphaMissense annotator there is a drop down for Class that would indicate ambiguous, likely_benign, and likley_pathogenic. If you are using the ClinGen Gene annotator there is a searchable column for Classification that indicates clinical validity classification. 

Please let me know if you need more help and thank you for using OpenCRAVAT!

**Comment by DogCatSpace** [comment]
Many thanks for the rapid reply. Two more questions:

 

1.	What other modules have something close to a drop down for Class that would indicate ambiguous, etc.?
2.	What is the meaning of this error message “C:\Program Files\OpenCRAVAT\python\lib\site-packages\twobitreader\__init__.py:84: SyntaxWarning: "is" with a literal. Did you mean "=="? elif x is 2:”

 

Bill

**Comment by jasminebro** [comment]
Hi @DogCatSpace. The ClinVar annotator displays Clinical Significance in a column where you can sort or filter (
Benign, Likely benign, Uncertain significance, Likely pathogenic, Pathogenic, Drug response, Association, Risk factor, Protective, Affects, Conflicting data from submitters, Other and Not provided). When you say "ambiguous" do you mean conflicting data or uncertain significance? 

Also, for the error would you mind telling me what version of Python you are using, what version of OpenCRAVAT you are using and when you are seeing the error on your screen? I can get our IT team to investigate the issue better with that information. Has this error hindered you from using the software?

Thank you!

Jasmine

**Comment by DogCatSpace** [comment]
Thanks.  Cravat version 2.9.1.  Python was installed with Cravat via the Cravat install so assuming it is the latest.
